jekyll/_cci2/orb-author-intro.md

```diff
@@ -6,90 +6,59 @@ description: "Starting point for how to authoring an orb"
+## Introduction to Orb Authoring
 
-[Orbs]({{ site.baseurl }}/2.0/orb-intro/) are made available for use in a configuration through the `orbs` key in the top level of your 2.1 [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.
+[Orbs]({{site.baseurl}}/2.0/orb-intro/) are [reusable packages of CircleCI configuration]({{site.baseurl}}/2.0/reusing-config/), enabling you to create parameterized commands, jobs, and executors that can be [used across multiple projects]({{site.baseurl}}/2.0/orb-concepts/).
 
-If you have determined that using an existing CircleCI or partner-certified orb will not address your specific workflow or job, then you may choose to author your own orb. Although more time-consuming than using an existing orb from the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/), authoring your own orb will enable you to create a pre-packaged configuration that suits your particular workflow. These pages will walk you through the steps required to author an orb, validate its configuration, and then publish this newly-created orb to the CircleCI Orb Registry.
+Hundreds of orbs are available from the [Orb Registry](https://circleci.com/orbs/registry/). Existing orbs are developed and maintained by a variety of authors: CircleCI, our technology partners, and the open-source community. Search the registry for the technology or service you would like to integrate with your projects.
 
-## Key Concepts
+If you want to join the community and author your own orb, the documents in this section are for you. If you manage multiple similar projects, consider abstracting out your config with orbs.
 
-Before authoring an orb, you should first familiarize yourself with some basic core concepts of orbs and how they are structured and operate. Gaining a basic understanding of these core concepts will enable you to write fully-featured orbs you can leverage and use in your workflows.
+Practice with [inline orbs](https://circleci.com/docs/2.0/orb-author/#writing-inline-orbs). Inline orbs can be defined within a single config file for easy and quick testing.
 
-Orbs consist of the following three elements:
+Orb Authors agree to our [Code Sharing Terms of Service](https://circleci.com/legal/code-sharing-terms/). All published orbs are made available publicly on the Orb Registry under the [MIT License agreement](https://opensource.org/licenses/MIT). For more information, see [Orb Licensing](https://circleci.com/orbs/registry/licensing).
 
-- Commands
-- Jobs
-- Executors
-
-### Commands
+## Key Concepts
 
-Commands are reusable sets of steps that you can invoke with specific parameters within an existing job. For example, if you want to invoke the command `sayhello`, you would pass the parameter `to` as follows:
+Orbs take [reusable configuration](({{site.baseurl}}/2.0/reusing-config/)) and package it in a way that can be published to the [Orb Registry]() so that they may be imported in multiple configuration files.
```

Comment:
```suggestion
Orbs take [reusable configuration]({{site.baseurl}}/2.0/reusing-config/) and package it in a way that can be published to the [Orb Registry]() so that they may be imported in multiple configuration files.
```